### PR TITLE
Add 'people to contact' to project template

### PR DESCRIPTION
**Pull request by jpkrohling**
This PR proposes a new section to the project template, requesting project proponents to identify the relevant people in the industry for this SIG. This would help us map where potential gaps are, and document who was invited to collaborate but refused. I believe this is very important to keep the new SIG process transparent.

Signed-off-by: Juraci Paixão Kröhling <juraci@kroehling.de>

### CODE REVIEW

**Review by svrnm — APPROVED** [review]
Good addition! LGTM!

**Review by maryliag — APPROVED** [review]

**Review by arminru — APPROVED** [review]

File: project-template.md [hunk 1]
```diff
@@ -20,6 +20,10 @@ In general, OTEPs are not accepted unless they come with working prototypes avai
 
 Who is currently planning to work on the project? If a project requires specialized domain expertise, please list it here. If a project is missing a critical mass of people in order to begin work, please clarify.
 
+### People to contact
+
+Who (people, companies) in the industry should be aware of this effort? Was there an attempt to get them onboard? What did they say?
```
Comment (danielgblanco) [inline project-template.md:25]: Once the explanatory text is removed, "People to contact" may look like a list of people to contact within the SIG if someone has a question, which I don't think is the purpose of this change.

What are your thoughts on something like "Industry outreach" or "Consultation and feedback"?

Comment (jpkrohling) [reply project-template.md:25]: Great point. I'll rename the section.

**Comment by jpkrohling** [comment]
PR updated, sorry for taking so long to work on the feedback.

**Review by mx-psi — APPROVED** [review]

**Pull request merged**